WcaOnRails/db/migrate/20170316030443_add_big_json_to_competitions.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+class AddBigJsonToCompetitions < ActiveRecord::Migration[5.0]
+  def change
+    add_column :Competitions, :big_json, :text
```

Comment:
"Cache" is an awkward term I could have improved on.

What I mean is: If the Competition itself changes, does the JSON have to change with it?

Or is this entirely separate data coming from the outside?